src/chapters/razor-views/create-template.rst

```diff
@@ -0,0 +1,128 @@
+   projects. Do NOT skip this practice, because the end of chapter exercises
+   pick up where the tutorials end.
+
+.. TODO: Add static view video. 
+   .. topics covered: Demonstrate how templates are already being returned in homecontroller, 
+   creating simple form template, updating controller to return view template
+
+YOUTUBE VIDEO HERE
+
+.. admonition:: Note
+
+   The starter code for this video is found at the `attribute-routing branch <https://github.com/LaunchCodeEducation/HelloASPDotNETDemo/tree/attribute-routing>`__
+   of ``HelloASPDotNETDemo``. The final code presented in this 
+   video is found on the `views-static branch <https://github.com/LaunchCodeEducation/HelloASPDotNETDemo/tree/views-static>`__.
+
+The sections below outline the main ideas presented in the video. However, the
+text is NOT a substitute for completing the work described in the clip.
+
+.. index:: ! View() method
+
+Controllers Return View Templates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Out of the box, your MVC project contains several Razor templates within the ``Views`` directory.
+The sub-directory ``Home``, found inside of ``Views``, contains the Razor templates returned from the 
+``HomeController`` action methods. Just as an action method's name will map to a route with the same 
+name, an action method's name can also correspond to a Razor template's name. The ``Index()`` method 
+inside of ``HomeController`` returns the ``Index.cshtml`` template found inside of ``Views/Home``.
+
+In order to return that template, the action method calls a **View() method**. The ``View()`` method finds 
+a template that's associated with the particular controller and action method that it is called from. 
+
+To override the default behavior of the ``View()`` method, you can pass in a parameter which is the name 
+of the template you want to render. For example, if we want an action method named ``Form()`` to return a 
+Razor template named ``WelcomeForm.cshtml``, we have ``Form()`` return ``View("WelcomeForm");``.
+
+Add a Template
+^^^^^^^^^^^^^^
+
+To create a template, we update ``HelloController.Index()`` to return a Razor template instead of a 
+string of HTML. In ``Views``, create a new subdirectory called ``Hello``.
+
+Within ``Hello`` create a new file of the *Razor View* file template. 
+
+**WINDOWS USERS:** Right click on the ``Views`` directory or any subdirectory and select *Add -> New Item*. 
+Avoid selecting *Add -> View* from your options. 
+
+**MAC USERS:** Right click on the ``Views`` directory or any subdirectory and select *Add -> New File*.
+
+**ALL USERS:** In the modal window that appears, look for the item labelled *Razor View*. 
+
+.. figure:: figures/razor-view-template-selection.png
+   :alt: User selects the *Razor View* file template.
+
+   Select the *Razor View* file template.
+
+
+Add the form HTML to the template.
+Update the ``HelloController`` method ``Index()`` to return ``View()``. 
```

Comment:
I would expand on these sentences. "Add the HTML form from the ``Index()`` method to your new template. Once you are done with that, update the ``Index()`` method so that it only returns ``View()``." When I first read this section, I missed what was going on here. Some learners who really count on the written text to drive home their understanding may also miss it.